### PR TITLE
FIT: Large FITRaw upgrade

### PR DESCRIPTION
**Pull request by afurs**

### CODE REVIEW

File: DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/ChannelData.h [hunk 1]
```diff
@@ -15,19 +15,20 @@
 #ifndef _FT0_CHANNELDATA_H_
 #define _FT0_CHANNELDATA_H_
 
+#include <Framework/Logger.h>
```
Comment (AllaMaevskaya) [inline DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/ChannelData.h]: Why did your add this line? We don't print anything in header. And similar remark about DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h

Comment (afurs) [reply DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/ChannelData.h]: @AllaMaevskaya check srcs

Comment (AllaMaevskaya) [reply DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/ChannelData.h]: so move include in srcs

Comment (afurs) [reply DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/ChannelData.h]: Ok

**Review by AllaMaevskaya — DISMISSED** [review]

File: DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/LookUpTable.h [hunk 2]
```diff
@@ -228,10 +228,9 @@ class LookUpTable
     std::cout << "lut_data.size " << lut_data.size() << std::endl;
     return o2::ft0::LookUpTable{lut_data};
   }
-  bool isTCM(int link, int ep)
-  {
-    return getChannel(link, 1, ep) == TCM_channel;
-  }
+  bool isTCM(int link, int ep) const { return getChannel(link, 1, ep) == TCM_channel; }
+  Topo getTopoPM(int globalChannelID) const { return mTopoVector[globalChannelID]; }
+  Topo getTopoTCM() const { return mTopoVector[TCM_channel]; }
```
Comment (shahor02) [inline DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/LookUpTable.h]: do you really want to return a copy or the reference would be enough?

**Review by shahor02 — CHANGES_REQUESTED** [review]
Also, CI produces lot of errors
````
/mnt/mesos/sandbox/sandbox/o2-fullci/sw/SOURCES/O2/6152/0/Detectors/FIT/raw/include/FITRaw/DigitBlockBase.h:229:25: error: statement should be inside braces [readability-braces-around-statements]
````

File: DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/LookUpTable.h (hunk 2 above)
Comment (afurs) [reply DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/LookUpTable.h]: @shahor02 I removed this lines. Code is fixed, CI seems to be okay.

**Review by shahor02 — APPROVED** [review]

**Pull request merged**